proposals/2021-01-delay-stargate-upgrade/README.md

```diff
@@ -0,0 +1,12 @@
+# Delay of Hub Stargate Upgrade
+
+The Stargate team is recommending that the Cosmos Hub reschedule the next upgrade to a new commit hash. The new commit hash is expected to be available on Tuesday Jan 26th with a new upgrade proposal immediately after.
+
+This governance proposal will signal that [proposal 35](https://www.mintscan.io/cosmos/proposals/35) will not be executed. The Hub governance will vote on forthcoming proposal aiming for a final upgrade on Feb 11th.
```

Comment:
Should we instead have all validators in agreement vote no on proposal 35, and have this PSA be broadcasted separately? Or is there a desire to have this be broadcasted via governance to ensure visibility by all relevant stakeholders?

Reply:
Put it on governance to ensure visibility